DXRando/DeusEx/Classes/Player.uc

```diff
@@ -177,6 +183,9 @@ function bool DXReduceDamage(int Damage, name damageType, vector hitLocation, ou
 
     bReduced = False;
     newDamage = Float(Damage);
+    if( newDamage > 49/CombatDifficulty ) {
+        newDamage = 49/CombatDifficulty;
+    }
```

Comment:
here we cap the maximum damage in a shot to 49 regardless of difficulty, shots to the player's head or torso get double damage after this so that would be 98 damage